src/forms/button/Button.tsx

```diff
@@ -70,14 +70,14 @@ export class Button extends React.PureComponent<ButtonProps> {
     }
 
     render() {
-        const {disabled, children, onClick, onSubmit, submit, icon, show} = this.props;
+        const {disabled, children, onClick, onSubmit, icon, show, type} = this.props;
```

Comment:
let's add lines
const {
disabled,
....,
} = this.props;

cause the line becomes too long

Reply:
and diff becomes less readable. When you add a new line you can see the type more quicker

Reply:
yes. I agree.